rest-api/src/test/scala/za/co/absa/enceladus/rest_api/integration/repositories/RunRepositoryIntegrationSuite.scala

```diff
@@ -42,6 +44,7 @@ class RunRepositoryIntegrationSuite extends BaseRepositoryTest {
   private val runFixture: RunFixtureService = null
 
   @Autowired
+  @Qualifier("runMongoRepository") // to correctly wire V2 runMongoRepository
```

Comment:
Another education request, please.

Reply:
This is the same case as https://github.com/AbsaOSS/enceladus/pull/2073#discussion_r897672416
A field of type `RunMongoRepository` can be wired with more than one class, so the qualifier here helps to resolve that -- when the field of this type is named `runMongoRepository`